reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java

```diff
@@ -264,6 +264,16 @@ final void drainReceiver() {
 				}
 
 				try {
+					if (logLeakDetection.isDebugEnabled()) {
+						if (v instanceof ByteBuf) {
+							((ByteBuf) v).touch(format(channel, "Bounded receiver " + a.getClass().getName() +
+									" will handle the message from this point"));
+						}
+						else if (v instanceof ByteBufHolder) {
+							((ByteBufHolder) v).touch(format(channel, "Bounded receiver " + a.getClass().getName() +
+									" will handle the message from this point"));
```

Comment:
"Bound receiver" perhaps you mean, or maybe just "Receiver".

Reply:
fixed